recipe/meta.yaml

```diff
@@ -24,8 +24,8 @@ requirements:
   run:
     - python
     - atom
-    - dask
-    - distributed
+    - dask <=2021.4.0
+    - distributed <=2021.4.0
```

Comment:
Did not the strict pin work? I am OK with `<=`, just curious.

Reply:
I didn't wait for CI to complete. It would probably work.